ui/components/ui/button/button.component.js

```diff
@@ -43,7 +43,7 @@ const Button = ({
   // we know to be erroneous attributes for a link. We will likely want to extract Link
   // to its own component in the future.
   let Tag = 'button';
-  if (type === 'link') {
+  if (type === 'link' || type === 'inline') {
     Tag = 'a';
   } else if (submit) {
```

Comment:
@NiranjanaBinoy : above can be breaking change for existing use of button as `inline`.

Reply:
@jpuri I did search for any previous usage of `<Button type='inline'/>` in our code, but I did not find any yet.

Reply:
Also, I am not sure is it ok to convert inline buttons into anchor tag. cc @georgewrmarshall

Reply:
My two cents is that this isn't a button. It shouldn't be styled like one – in fact there is some padding around the link which is probably not what we want. It also doesn't act like one — i.e. it doesn't change the state of the application in any way. So I would think an `<a>` element is more appropriate instead of Button. But curious to know what others think.

Reply:
`type="link"` is the correct prop to use here as the implementation is using `href` attributes so it's rendering semantically correct html(`<a href="...">`). But it looks like the styles are off. I would have expected the styles `font-size: inherit; display: inline-flex; padding: 0;` to be added but it's retaining the button display and padding. We also don't want to add inline here as there are instances where we need it to look like a link but semantically act as a button e.g. using `onClick`.

My suggestion would be to remove the `inline` type as I think it does exist [see here](https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/ui/button/button.component.js#L23)(you may need to rebase from develop) and add the appropriate custom styles to each instance of the link type buttons using a `className`. e.g. `display: inline; font-size: inherit; padding: 0` or whatever it may be.

Reply:
Moving forward it would be nice if we could include a polymorphic prop to all of our components so we can change the rendered html to whatever is necessary independently of the style type...
e.g.
```
<Button href="http://..." as="a" type="inline" >Click me</Button>

// Renders HTML
<a href="http://..." as="a" type="inline" >Click me</a>
```